src/app/app.tsx

```diff
@@ -24,19 +24,23 @@ import { Title } from '../styles/components/Title'
 import { Form } from '../styles/components/Input'
 import { LabeledCheckbox } from './components/Checkbox'
 
-import { Result, SubmitEvent, getElementValues } from './features/login'
+import { Status, SubmitEvent, getElementValues } from './features/login'
 import Footer from './components/Footer'
 
 export default ({ optOutUrl }: OptOutUrl) => {
   const {
-    state: { isWrong, isKeepLogin },
+    state: { status, isKeepLogin },
     dispatch,
   } = useLoginReducer()
 
+  const isWrong = [Status.invalid, Status.blocked].includes(status)
```

Comment:
```suggestion
  const isWrong = status === Status.invalid || status === Status.blocked
```

이 부분에서 Array.prototype.includes가 쓰일 이유가 없을 것 같아요. 좀 성급한 최적화 같긴 하지만 약간의 오버헤드도 있을 것 같은데, 명확성 측면에서도 직접 비교가 유리하지 않을까요?